viam-cartographer.go

```diff
@@ -35,29 +32,28 @@ import (
 	rdkutils "go.viam.com/rdk/utils"
 	"go.viam.com/rdk/vision"
 	slamConfig "go.viam.com/slam/config"
-	"go.viam.com/slam/dataprocess"
+	"go.viam.com/slam/sensors/lidar"
 	slamUtils "go.viam.com/slam/utils"
 	goutils "go.viam.com/utils"
 	"go.viam.com/utils/pexec"
 	"go.viam.com/utils/protoutils"
-)
 
-var (
-	// Model is the model name of cartographer.
-	Model                         = resource.NewModel("viam", "slam", "cartographer")
-	cameraValidationMaxTimeoutSec = 30 // reconfigurable for testing
-	dialMaxTimeoutSec             = 30 // reconfigurable for testing
+	dim2d "github.com/viamrobotics/viam-cartographer/internal/dim-2d"
 )
 
+// Model is the model name of cartographer.
+var Model = resource.NewModel("viam", "slam", "cartographer")
+
 const (
-	// DefaultExecutableName contains the name of the cartographer grpc server executable.
-	DefaultExecutableName       = "carto_grpc_server"
-	defaultDataRateMsec         = 200
-	defaultMapRateSec           = 60
-	cameraValidationIntervalSec = 1.
```

Comment:
Good catch! That was indeed a typo. I double-checked all values again and renamed the variables in the tests with a `Test...` prefix to distinguish them more from the default values and to make it clear that they're set to these values specifically for the tests.

Reply:
Sounds great! Very minor optional nit - we could name the variables xxxForTest, since TestXxx has some ambiguity with whether you're testing Xxx, or if Xxx is meant to be used in tests